issue_templates.md

```diff
@@ -0,0 +1,144 @@
+# Feature Request
+name: Feature Request
+
+about: Suggest an idea for this project
+
+title: "[FEATURE]"
+
+labels: 'feature request'
+
+**Is your feature request related to a problem? Please describe.**
+
+[A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] ]
+
+**Describe the solution you'd like**
+
+[A clear and concise description of what you want to happen.]
+
+**Describe alternatives you've considered**
+
+[A clear and concise description of any alternative solutions or features you've considered.]
+
+**Additional context**
+
+[Add any other context or screenshots about the feature request here.]
+
+# Bug Report
+name: Bug Report
+
+about: Create a report to help us improve
+
+title: "[BUG]"
+
+labels: 'bug'
+
+
+**Describe the bug**
+
+[A clear and concise description of what the bug is.]
+
+**To Reproduce**
+
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+
+[A clear and concise description of what you expected to happen.]
+
+**Screenshots**
+
+[If applicable, add screenshots to help explain your problem.]
+
+**Environment (please complete the following information):**
+- OS: [e.g. Fedora, RHEL, CentOS, Ubuntu, Debian, Windows, MacOS, Linux]
+- Browser [e.g. chrome, safari]
+- Version [e.g. 22]
+
+**Additional context**
+
+[Add any other context about the problem here.]
+
+
+# Compatibility Issue
+
+name: Compatibility Issue
+
+about: Report a problem related to system or browser compatibility
+
+title: "[COMPATIBILITY]"
+
+labels: 'compatibility'
+
+
+
+**Describe the Compatibility Issue**
+
+[A clear and concise description of what the compatibility issue is.]
+
+**To Reproduce**
+
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Environment (please complete the following information):**
+- OS: [e.g. Windows, MacOS, Linux]
+- Browser [e.g. chrome, safari, edge]
+- Version [e.g. 22]
+
+**Expected behavior on this environment**
+
+[A clear and concise description of what you expected to happen]
+
+**Screenshots**
+
+[If applicable, add screenshots to help explain your problem]
+
+**Additional context**
+
+[Add any other context about the problem here.]
+
+
+# Improvement Suggestions
+name: Improvement Suggestions
+
+about: Suggest a way to improve something that already exists in the project
+
+title: "[IMPROVEMENT]"
+
+labels: 'improvement'
+
+
+**Describe the current behavior**
+[A clear and concise description of what the behavior is.]
+
+**Describe the improvement you'd like**
+[A clear and concise description of what you want to happen.]
+
+**Additional context**
+[Add any other context or screenshots about the improvement suggestion here.]
+
+
+
+# Question
```

Comment:
Wouldn't this be a better fit for the 'Discussions' feature on GitHub?

Reply:
https://docs.github.com/en/discussions/quickstart

Good point. You may need to set it up in the setting of this repo

@Cali0707